client/layout/guided-tours/docs/API.md

```diff
@@ -0,0 +1,223 @@
+
+### Props
+
+* `name`: (string) Unique identifier of the step, used for addressing a step from `Next` or `Continue`.
+* `target`: (string, optional) Target which this step belongs to and will be used for positioning. Value of this prop is used to look up according `[data-tip-target]` in DOM. If you start this value with `.` (dot), it will be evaluated as a query selector, so you can select elements that have no `[data-tip-target]` defined.
+* `placement`: (string, optional) Placement. Possible values: 'below', 'above', 'beside', 'center', 'middle', 'right'
+* `arrow`: (string, optional) If defined, step will get arrow pointing to a direction. Available: 'top-left', 'top-center', 'top-right',
+'right-top', 'right-middle', 'right-bottom', 'bottom-left', 'bottom-center', 'bottom-right', 'left-top', 'left-middle', 'left-bottom'
+* `style`: (object, optional) Will be used as step's inline style. Use sparingly and with caution for minor tweaks in positioning or z-indexes and avoid changing the look and feel of Guided Tours. If you use this is a way that could benefit all Guided Tours globally, please consider creating an issue.
+* `when`: (function, optional) This is a redux selector that can prevent step from showing when it evaluates to false. Define `next` prop to tell Guided Tours name of the step it should skip to. If you omit this prop, step will be rendered as expected.
+* `next`: (string, optional) Define this to tell Guided Tours name of the step it should skip to when `when` evaluates to false.
+* `children`: (nodes) Content of step. Usually a paragraph of instructions and some controls for tour. See below for available options.
+
+### Example
+
+```jsx
+// single step tour
+<Tour path="/me" name="uselessTour">
+  <Step>
+    <p>This is a page about you.</p>
+    <ButtonRow>
+      <Quit />
+    </ButtonRow>
+  </Step>
+</Tour>
+```
+
+Note that all text content needs to be wrapped in `<p>` so it gets proper styling.
+
+For more comprehensive examples, look at [TUTORIAL.md](TUTORIAL.md) or explore existing tours in client/layout/guided-tours/tours.
+
+## ButtonRow
+
+ButtonRow is a React component to display button controls in Step and takes care of their proper styling. Usually used as the last child of Step to contain all available controls.
+
+### Example
+
+```jsx
+<Step>
+  <p>ButtonRow Example</p>
+  <ButtonRow>
+    <Next step="next-step" />
+    <Quit />
+  </ButtonRow>
+</Step>
+```
+
+## Continue
+
+Continue is a React component that you can use in Step to programmatically continue the tour to other step based on user interaction with Calypso.
+
+### Props
+
+* `step`: (string) Name of the step the tour will advance to.
+* `target`: (string, optional) Name of `[data-tip-target]` that would be watched.
```

Comment:
I think we might want to extract docs for `targetForSlug` somehow and link to it from several places, where we explain `target`